integration-tests/go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/smartcontractkit/chainlink-solana/integration-tests
 
 go 1.21.7
 
+replace github.com/smartcontractkit/tdh2/go/ocr2/decryptionplugin => github.com/jmank88/tdh2/go/ocr2/decryptionplugin v0.0.0-20240321204202-bf6e0344b64b
```

Comment:
what is this?

Reply:
I didn't have auth to push a branch to tdh2 so this was required to use my personal fork of that repo until the PR is merged.